recipes/default.rb

```diff
@@ -16,6 +16,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+if platform_family?('rhel')
+  include_recipe 'osl-selinux'
+end
```

Comment:
This is a cleaner way to do it. Please fix it globally.
```suggestion
include_recipe 'osl-selinux' if platform_family?('rhel')
```